liquid.theme

```diff
@@ -24,6 +24,7 @@ if [[ "$(locale -k LC_CTYPE | sed -n 's/^charmap="\(.*\)"/\1/p')" == *"UTF-8"* ]
     LP_MARK_STASH="+"          # if git has stashs
     LP_MARK_SHORTEN_PATH=" … " # prompt mark in shortened paths
     LP_MARK_PERM=":"           # separator between host and path
+    LP_MARK_KUBECONTEXT=""     # Kubernetes context
```

Comment:
If you are sticking with the empty string as the default mark, I think the changes to this file should be removed. This file is an example, and it does not need every new feature added.